snapcraft/plugins/rust.py

```diff
@@ -54,6 +54,12 @@ def schema(cls):
 
     def __init__(self, name, options, project):
         super().__init__(name, options, project)
+        self.build_packages.extend([
+            'build-essential',
+            'git',
+            'curl',
+            'file',
```

Comment:
These are the things that were required for me to get rustc installed on one of the launchpad builders. Can't find specific documentation about it but that's where this list came from.